Remotion/Web/Development.WebTesting/WebDriver/Configuration/Chrome/ChromeBinariesProvider.cs

```diff
@@ -155,14 +180,11 @@ private string GetChromeDriverVersion (string chromeVersion)
       }
     }
 
-    private bool ChromeDriverExists (string chromeDriverVersion)
+    private string GetChromeDriverTempPath (string chromeDriverVersion, string? zipArchiveFolder)
     {
-      return File.Exists(GetChromeDriverTempPath(chromeDriverVersion));
-    }
-
-    private string GetChromeDriverTempPath (string chromeDriverVersion)
-    {
-      return Path.Combine(GetChromeDriverTempDirectory(chromeDriverVersion), c_driverExecutableName);
+      return zipArchiveFolder != null
+        ? Path.Combine(GetChromeDriverTempDirectory(chromeDriverVersion), zipArchiveFolder, c_driverExecutableName)
+        : Path.Combine(GetChromeDriverTempDirectory(chromeDriverVersion), c_driverExecutableName);
```

Comment:
```suggestion
      return zipArchiveFolder != null
          ? Path.Combine(GetChromeDriverTempDirectory(chromeDriverVersion), zipArchiveFolder, c_driverExecutableName)
          : Path.Combine(GetChromeDriverTempDirectory(chromeDriverVersion), c_driverExecutableName);
```
I think this has two indents, not one, since it's a statement line wrap.